smarsy/bs_helper.py

```diff
@@ -0,0 +1,45 @@
+from bs4 import BeautifulSoup
+
+
+class BSHelper(object):
+    """
+     The help class for BeautifulSoup library
+     """
+    def __init__(self, html):
+        self.html = html
+
+    @property
+    def get_bs_object(self):
+        """
+        Utility funtcion:
+            - Accepts html and checks its validity using BeautifulSoup library,
+            return BS object or False
+        """
+        try:
+            soup = BeautifulSoup(self.html, 'html.parser')
+        except TypeError:
+            return False
+        return soup
+
+    def bs_safe_select(self, html, *args):
+        """
+        Utility function used to get a content string from a
+        HTML and tuple of selectors. Returns False
+        if no object is found for the given selector
+        """
+        for arg in args:
+            selectedElems = html.select_one(arg)
```

Comment:
won't it always take the last output of the `select_one` ? It's not adding, for every iteration it re-assigns `selectedElems` with the new value. No? or is it expected?

Reply:
That is what is expected. Each new iteration overrides a variable selectedElems

Reply:
then I don't get it, does it supposed to return:
* all objects for all found selectors?
* the last found object?
* the first found object?
* any object?
* is it expected to be some kind of chaining action, when the result of the previous iteration is used in the next one?

Reply:
> then I don't get it, does it supposed to return:

we take the object, apply the method select_one with selector 1 to it, then apply the method select_one with selector 2 to the received object, then apply the method select_one with the selector X to the received object and return the object or False

Reply:
could you please provide real example from smarsy website? expected call with expected result?

Reply:
From issue#51
`<TD valign=top align="left" width="120"><img src="https://smarsy.ua/images/mypage/parent_1.png"></TD>`
We must find td with valign=top and in received object find img[src]

Reply:
and what would be the function call for that html?

Reply:
`html.select_one([valign=top]).select_one('img[src]')`

Reply:
will it work? Additionally here you are passing single value parameter, but in your function you are expecting array. Please provide an example with array